openmetadata-ui/src/main/resources/ui/src/pages/ServiceDetailsPage/ServiceDetailsPage.tsx

```diff
@@ -1056,6 +1057,18 @@ const ServiceDetailsPage: FunctionComponent = () => {
         isHidden: !servicePermission.EditAll,
         key: EntityTabs.CONNECTION,
         children: testConnectionTab,
+      },
+      {
+        name: t('label.announcement-plural'),
+        key: EntityTabs.ANNOUNCEMENT,
+        count: 0,
```

Comment:
We need to replace it with actual count right?

Reply:
yes, that will be replaced when the backend changes are available for total announcements count in counts api call. Currently we are not fetching all the announcements (active  & inactive  both) all at once. Fetching only the selected announcement type ie. active or inactive.